lib/hit/arkit_node_pan_result.dart

```diff
@@ -16,6 +20,10 @@ class ARKitNodePanResult {
   @Vector2Converter()
   final Vector2 translation;
 
+  // The touch location relative to the scene view
```

Comment:
```suggestion
  /// The touch location relative to the scene view
```
let's make it a documentation line instead of just a comment.